lib/modules/platform/azure/index.spec.ts

```diff
@@ -316,6 +316,76 @@ describe('modules/platform/azure/index', () => {
       expect(res).toMatchSnapshot();
     });
 
+    it('returns pr if found matches targetBranch', async () => {
+      azureApi.gitApi.mockImplementationOnce(
+        () =>
+          ({
+            getPullRequests: jest
+              .fn()
+              .mockReturnValue([])
+              .mockReturnValueOnce([
+                {
+                  pullRequestId: 1,
+                  sourceRefName: 'refs/heads/branch-a',
+                  targetRefName: 'refs/heads/branch-b',
+                  title: 'branch a pr',
+                  status: PullRequestStatus.Active,
+                },
+                {
+                  pullRequestId: 2,
+                  sourceRefName: 'refs/heads/branch-a',
+                  targetRefName: 'refs/heads/branch-c',
+                  title: 'branch a pr',
+                  status: PullRequestStatus.Active,
+                },
+              ]),
+            getPullRequestCommits: jest.fn().mockReturnValue([]),
+          } as any)
+      );
+      const res = await azure.findPr({
+        branchName: 'branch-a',
+        prTitle: 'branch a pr',
+        state: 'open',
+        targetBranch: 'branch-c',
+      });
+      expect(res).toMatchSnapshot();
```

Comment:
use `toEqual` or `toMatchObject`

Reply:
all the other existing tests for `findPr` are using `toMatchSnapshot`.  I either keep it consistent or update them all

Reply:
You can update them all

Reply:
updated